src/components/swap/AdvancedSwapDetails.tsx

```diff
@@ -75,7 +75,7 @@ export function AdvancedSwapDetails({ trade, allowedSlippage, syncing = false }:
             }
           >
             <ThemedText.BodySmall color="textSecondary">
-              <Trans>Minimum output</Trans>
+              <Trans>{trade.tradeType === TradeType.EXACT_INPUT ? 'Minimum output' : 'Maximum input'}</Trans>
```

Comment:
so i think our Translation library doesn't work super well when there's a ternary inside - can you amend to do the switch outside of the `<Trans>`? otherwise lgtm and thanks for this fix :)

Reply:
Fixed!